src/components/SpotlightCard/SpotlightCard.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState, useEffect } from "react";
+import "./SpotlightCard.scss";
+import TextLink from "../TextLink";
+
+export interface SpotlightCardProps {
+    title: string;
+    imageSource?: string;
+    imageAltText?: string;
+    subtitle?: string;
+    description?: string;
+    textLink?: string;
+    halfWidth?: boolean;
+    // TODO add CTA button when access to CFA repo is granted
+}
+
+const SpotlightCard = (props: SpotlightCardProps) => {
+    const [expanded, setExpanded] = useState(false);
+    const [description, setDescription] =  useState('');
+    const [showReadToggle, setShowReadToggle] = useState(false);
+    const [readToggleText, setReadToggleText] = useState('');
+
+    const textLinkText = 'Meet the Leadership Team';
+    const substringLength = 385;
+    const mobile = window.innerHeight <= 767 && window.innerWidth <= 767;
```

Comment:
I wonder if we really need to check the screen size or if we could achieve the same goal with just CSS. It's less expensive. Can you try it out?

Reply:
The issue with only doing CSS is that if the halfWidth parameter is ever passed in, and it is true, then its CSS styling will applied be no matter what, even when we go into the mobile width. This doesn't cause any problems at the moment, but I think this should be avoided so that the rendered code is more accurate. I wasn't able to come up with any way to do CSS besides this method, do you have any ideas?